packages/tilda/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/lanoxx/tilda"
+TERMUX_PKG_DESCRIPTION="A Gtk based drop down terminal for Linux and Unix"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@ELWAER-M"
+TERMUX_PKG_VERSION="1.5.0"
+TERMUX_PKG_SRCURL="https://github.com/lanoxx/tilda/archive/refs/tags/tilda-${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="f664c17daca2a2900f49de9eb65746ced03c867b02144149ef21260cbcd61039"
+TERMUX_PKG_DEPENDS="glib, gtk3, libvte, libconfuse, libx11, gettext, automake, autoconf"
+TERMUX_PKG_EXTRA_MAKE_ARGS="--silent"
+
+
+termux_step_pre_configure() {
+    NOCONFIFURE=1 ./autogen.sh
```

Comment:
Typo there. Should be `NOCONFIGURE=1 ./autogen.sh`, i.e. not `NOCONFIFURE=1 ...`.